test/09_contracts.ts

```diff
@@ -22,6 +22,23 @@ const clearDeviceTypeResource = () => {
 	});
 };
 
+const compareContractsToDb = async ({
+	contract,
+	db,
+}: {
+	contract: string;
+	db: string;
+}) => {
+	const fromContracts = (await getContracts(contract))
+		.map((x) => x.slug)
+		.sort();
+	const fromDb = (await supertest().get(`/resin/${db}`).expect(200)).body.d
+		.map((x: { slug: string }) => x.slug)
+		.sort();
+
+	expect(fromContracts).to.eql(fromDb);
```

Comment:
For the sake of 2 characters I'd find the full word more naturally readable:
```suggestion
	expect(fromContracts).to.equal(fromDb);
```

Reply:
I agree, I just used it cause I saw it in different places like this

Reply:
Ah I hadn't noticed it was moved code, either way I think the full word makes sense (I'd probably even find it quicker to type as well because of muscle memory)